rtt_roscomm/include/rtt_roscomm/rostopic.h

```diff
@@ -1,29 +1,34 @@
-/***************************************************************************
-  tag: Ruben Smits  Tue Nov 16 09:18:49 CET 2010  rtt_rostopic.h
-
-                        rtt_rostopic.h -  description
-                           -------------------
-    begin                : Tue November 16 2010
-    copyright            : (C) 2010 Ruben Smits
-    email                : first.last@mech.kuleuven.be
-
- ***************************************************************************
- *   This library is free software; you can redistribute it and/or         *
- *   modify it under the terms of the GNU Lesser General Public            *
- *   License as published by the Free Software Foundation; either          *
- *   version 2.1 of the License, or (at your option) any later version.    *
- *                                                                         *
- *   This library is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU     *
- *   Lesser General Public License for more details.                       *
- *                                                                         *
- *   You should have received a copy of the GNU Lesser General Public      *
- *   License along with this library; if not, write to the Free Software   *
- *   Foundation, Inc., 59 Temple Place,                                    *
- *   Suite 330, Boston, MA  02111-1307  USA                                *
- *                                                                         *
- ***************************************************************************/
+/*
+ * tag: Ruben Smits  Tue Nov 16 09:18:49 CET 2010  rtt_rostopic.h
+ *
+ *        This file is part of the OROCOS toolchain ROS project
+ *
+ *         (C) 2010 Ruben Smits, ruben.smits@mech.kuleuven.be
```

Comment:
Please remove the `tag:` line from all updated headers, not only the ones that have been modified in https://github.com/orocos/rtt_ros_integration/pull/154/commits/d620096ae4ab52e4ff93b1aab2dc65cd6d04bb4e. The information is redundant. In this particular case the file name is wrong anyway, because the header has been renamed a few years ago.

nit: Vertical whitespace is not consistent across different files, and I also propose to left-align all paragraphs after the asterisk boxes have been removed. The latter would make it easier to eventually update the copyright holder after the license header is copied to a new source or header file.